mlir/include/mlir/Conversion/Passes.td

```diff
@@ -125,6 +125,12 @@ def ArithToAMDGPUConversionPass : Pass<"convert-arith-to-amdgpu"> {
   }];
 
   let dependentDialects = ["amdgpu::AMDGPUDialect", "vector::VectorDialect"];
+
+  let options = [
+    Option<"saturateFP8Truncf", "saturate-fp8-truncf", "bool",
+           /*default=*/"false",
+           "Whether truncation to 8-bit float types should be saturating">,
```

Comment:
nit: suggestion, maybe `Use saturating truncation for 8-bit float types`? And prefix the option with `enable-` or `use-`?